<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistroRetribuciones</class>
 <widget class="QDialog" name="RegistroRetribuciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>420</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
   <property name="icon">
    <iconset resource="imagenes/imagenes.qrc">
     <normaloff>:/atras.png</normaloff>:/atras.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuardar">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="icon">
    <iconset resource="imagenes/imagenes.qrc">
     <normaloff>:/save.jpeg</normaloff>:/save.jpeg</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="listRetribuciones">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>351</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Listado de Retribuciones...</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>420</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnEliminar">
      <property name="text">
       <string>-</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAdd">
      <property name="text">
       <string>+</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>421</width>
     <height>351</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>401</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Denominación</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtDenominacion"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Cuantía</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtCuantia"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>144</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Código</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtCodigo"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>111</y>
      <width>211</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Clave</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbClave"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="imagenes/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
